Documentation/ceph-upgrade.md

```diff
@@ -265,7 +268,11 @@ Let's get started!
 > instructions to [migrate the Drive Group spec](#migrate-the-drive-group-spec) before performing
 > any of the upgrade steps below.
 
-## 1. Update common resources and CRDs
+## 1. Refer to the updates section for ancillary resources
+If you are using additional resources, refer to their individual upgrade guides as needed.
+- [Prometheus monitoring](ceph-monitoring.md#updates-and-upgrades)
```

Comment:
It's just a single yaml to update, right? How about if we just give them the command directly here in the upgrade guide and link them to the other doc in case they want more info.